src/components/Sidebar/index.js

```diff
@@ -42,26 +38,19 @@ const Sidebar = () => {
                 </Nav>
             </Navbar>
             <div className="social-icons">
-                <LogoGithub
-                    height="25px"
-                    width="25px"
-                />
-                <LogoLinkedin
-                    height="25px"
-                    width="25px"
-                />
-                <Mail
-                    height="25px"
-                    width="25px"
-                />
-                <LogoInstagram
-                    height="25px"
-                    width="25px"
-                />
-                <LogoBehance
-                    height="25px"
-                    width="25px"
-                />
+                <a href="https://github.com/ayu-ayushisood" target="_blank">
+                    <LogoGithub className="icons-style" />
+                </a>
+                <a href="https://www.linkedin.com/in/ayushi-sood" target="_blank">
```

Comment:
Also, for all <a> tags, we should use `rel=noreferrer` for safety purposes

Reply:
okay. done